Framework/DataHandling/src/LoadInstrument.cpp

```diff
@@ -48,12 +49,14 @@ void LoadInstrument::init() {
                       "Workspace", "Anonymous", Direction::InOut),
                   "The name of the workspace to load the instrument definition "
                   "into. Any existing instrument will be replaced.");
+  const std::vector<std::string> extensions{".xml", ".nxs", ".hdf5"};
```

Comment:
Ultimately, should probably share implementations of this between this alg and `LoadEmptyInstrument`